components/textUtils/textUtilsPage.css

```diff
@@ -0,0 +1,25 @@
+.container__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.container__content__sub-title {
+    align-self: flex-start;
+    margin-top: 3rem;
+    margin-bottom: 2rem;
+}
+
+.container__code-preview {
+    width: 80rem;
+    min-height: 22rem;
+    margin-top: 5rem;
+    align-self: center;
+    justify-content: center;
+    overflow: scroll;
```

Comment:
just a suggestion. use overflow:auto this have scrollbar only when needed not always.

Reply:
Oh nice, changed it to auto.